docker-grinder-wrapper.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env sh
+
+export PS1="(grinder-docker) $PS1"
+
+host="0.0.0.0"
+port="5000"
+
+cd ./map/
+flask run --host=$host --port=$port >& /dev/null &
+flask_pid=$!
+cd ..
+
+echo -e "\e[1mWelcome to\e[0m"
```

Comment:
[Check this out](https://context.reverso.net/%D0%BF%D0%B5%D1%80%D0%B5%D0%B2%D0%BE%D0%B4/%D0%B0%D0%BD%D0%B3%D0%BB%D0%B8%D0%B9%D1%81%D0%BA%D0%B8%D0%B9-%D1%80%D1%83%D1%81%D1%81%D0%BA%D0%B8%D0%B9/welcome+to+the), maybe we need an article 'the' here?
![gif](https://media.giphy.com/media/3ohhwABskCbafiv5aU/giphy.gif)

Or we can remove this line at all, as you wish :wink:

Reply:
XD, ok

Reply:
Still not changed.